prov/efa/src/rxr/rxr_atomic.c

```diff
@@ -176,6 +181,14 @@ ssize_t rxr_atomic_generic_efa(struct rxr_ep *rxr_ep,
 		 * support DELIVERY_COMPLETE
 		 * by nature
 		 */
+
+		if (queue_tx) {
+			queue_tx_ctrl_pkt(rxr_ep,
```

Comment:
This function name is too simple, it should be ```rxr_ep_queue_tx_ctrl_pkt```.

Reply:
queueing logic no longer used. resolving.